edb/edgeql/compiler/casts.py

```diff
@@ -128,12 +128,11 @@ def compile_cast(
                 subctx.implicit_tid_in_shapes = False
                 viewgen.compile_view_shapes(ir_set, ctx=subctx)
         elif (orig_stype.issubclass(ctx.env.schema, json_t)
-              and new_stype.is_enum(ctx.env.schema)):
+              and new_stype.is_enum(ctx.env.schema) and False):
```

Comment:
```suggestion
              and new_stype.is_enum(ctx.env.schema)):
```

Reply:
oops! debug code slipped in the last update

Reply:
Was it caught by tests?

Reply:
Yes, tests failed.